forta-uniswap-event-py/src/config/agent-settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "uniswap_v2_router_addr": "0x7a250d5630b4cf539739df2c5dacb4c659f2488d",
+  "ether_threshold": 5000000000000000000,
```

Comment:
Would this be more user friendly as "5"? 
If not, I would change the key to "ether_threshold_wei".